changelog.d/implicit-rules-sc-core.added

```diff
@@ -0,0 +1 @@
+Changed `semgrep-core` so that it can now be run with `-rules` on `.yaml` files which do not have a top-level `rules: ...` key. This means you can now copy paste from the playground editor directly into a `.yaml` file for use with `semgrep-core`.
```

Comment:
Unless this means Semgrep CLI will also accept `.yaml` files without a top `rules:` element, this should not go into the changelog, since semgrep-core is internal only.